less/slideshow.less

```diff
@@ -22,10 +21,19 @@
           left: 0;
           bottom: 0;
           width: 100%;
-          text-align: center;
+          text-align: left;
           color: #fff;
-          background: rgba(0,0,0,0.4);
           margin: 0;
+          padding: 10px 10px 5px 10px;
+          background: rgba(0,0,0,0);
+          background: -moz-linear-gradient(top, rgba(0,0,0,0) 0%, rgba(0,0,0,0.6) 100%);
+          background: -webkit-gradient(left top, left bottom, color-stop(0%, rgba(0,0,0,0)), color-stop(100%, rgba(0,0,0,0.6)));
+          background: -webkit-linear-gradient(top, rgba(0,0,0,0) 0%, rgba(0,0,0,0.6) 100%);
+          background: -o-linear-gradient(top, rgba(0,0,0,0) 0%, rgba(0,0,0,0.6) 100%);
+          background: -ms-linear-gradient(top, rgba(0,0,0,0) 0%, rgba(0,0,0,0.6) 100%);
+          background: linear-gradient(to bottom, rgba(0,0,0,0) 0%, rgba(0,0,0,0.6) 100%);
```

Comment:
can we add this into a less variable?
also is `to` a misspell?

Reply:
@nistormihai to is not a misspell its for older version of almost all browsers.
I can create a less variable, but let's merge this, and I will create less variable after the release.